.known-issues/doc/kerneldoc/unknown_options.conf

```diff
@@ -0,0 +1,2 @@
+# since we only parse Kconfig under kernel/, need to ignore those under arch/
```

Comment:
This "known issues" filter will apply to all documentation builds and could let some errors slip by for normal doc builds.  Perhaps the errors should be removed by adding a few more Kconfig files as part of what we'd consider the "kernel" configurations.

Sure seems like features the kernel relies on such as FP and MPU configuration, could be considered part of the kernel docs.

Reply:
Both ```arch/Kconfig``` and ```kernel/Kconfig``` are included by ```Kconfig.zephyr``` where they are at the same level in the Kconfig hierarchical "tree". I can't think of a scalable solution without introducing another top level Kconfig file just for document generation. A hierarchy that works for kernel doc may not work for api doc. But adding more files means that someone needs to maintain them.

Another solution would be to parse all the relevant RST files to extra all the referenced Kconfig options. The list of options will be fed into the genrest.py script to generate the kconfig RST files. However, this is error prone as we have to parse RST files ourselves.

Of course, we can also not customize the Kconfig doc and includes everything. HTML would probably be fine (except with more files and linkes), but number of pages for PDF will increase dramatically.

Reply:
The latest version has a way to specify what the root Kconfig to include necessary options. So this is no longer an issue.